metabase/metabase
# Show newly created model as option in new question

Fixes #24878

### How to Test

1. Create a model
2. Reload the page
3. Create a question and turn it into a model
4. Without refreshing the browser, create a question and choose Models as the source.

Last created Model should be displayed as an option for the new question.
